monkey_island/cc/services/config.py

```diff
@@ -794,29 +797,42 @@
     }
 }
 
+ENCRYPTED_CONFIG_ARRAYS = \
```

Comment:
This is duplicated here (with schema information) and in  `add_system_info_creds_to_config`
Can we avoid this somehow?

Reply:
I'm not sure I know what you're talking about

Reply:
There are two seperate places in the code where what fields need to be encrypted are listed.

Reply:
These 2 places are (sort of) unrelated. It's true that currently the credentials are the only config values being encrypted. Maybe in the future we'll have further secrets we'd want to encrypt but not use in add_system_info_creds_to_config.

Reply:
Yeah, the problem is that keeping track of whats encrypted requires a double lookup. If we can't avoid it oh well

Reply:
Can't avoid it right now